packages/expo-doctor/src/utils/getRemoteVersionsForSdkAsync.ts

```diff
@@ -0,0 +1,26 @@
+import { getVersionsAsync } from '../api/getVersionsAsync';
+
+export type DependencyList = Record<string, string>;
+
+export const getRemoteVersionsForSdkAsync = async (
+  sdkVersion?: string
+): Promise<DependencyList> => {
+  const { sdkVersions } = await getVersionsAsync({ skipCache: true });
+  if (sdkVersion && sdkVersion in sdkVersions) {
```

Comment:
Why call the endpoint first, and _then_ check if sdkVersion is even passed in. Check `if (sdkVersion)` before calling the endpoint.